hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/Compactor.java

```diff
@@ -130,6 +130,8 @@ public CompactionProgress getProgress() {
     public int maxTagsLength = 0;
     /** Min SeqId to keep during a major compaction **/
     public long minSeqIdToKeep = 0;
+    /** Total file size of the compacted files **/
```

Comment:
'files' or 'file'? If first one, please change the field name. Otherwise, change the documentation.